dev/domain-controller/README.md

```diff
@@ -1,50 +1,88 @@
-# Create a complete deployment of the Cloud Access Manager solution including a Domain Controller
-
-This template will deploy 5 virtual machines (along with a new VNet, Storage Account, Load Balancer, Azure KeyVault, and Gateway).
-
-To administer the deployment through the Cloud Access Manager GUI, https: to the public IP of the applicationGateway1 Application Gateway.
-
-To connect to the pre-created Agent virtual machine, point the PCoIP client to the public IP of the applicationGateway1 Application gateway and login with the administrator credentials.
-
-To manage the Active Directory Domain, RDP to the public IP address of vm-dc (the domain controller).
-
-Click the button below to deploy
-
-<a target="_blank" href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fteradici%2Fdeploy%2Fmaster%2Fdev%2Fdomain-controller%2Fazuredeploy.json">
-    <img src="http://azuredeploy.net/deploybutton.png"/>
-</a>
+ ## What is Cloud Access Manager?
+ Teradici Cloud Access Manager (CAM) is a one click deployment solution that provides a level of brokering on top of existing CAS deployments. CAM will enable you to assign and revoke virtual machines to users as well as creating and destroying virtual workstations. The CAM solution consists of the following components:
```

Comment:
I think for parallel language we want 'create and destroy' rather than 'creating and destroying'?